packages/core/logger/src/index.js

```diff
@@ -197,13 +197,13 @@ export class Logger {
     this.writeToFile.push({ args, origin, prefix: "[trace]" });
   }
 
-  dir(obj) {
-    if (!obj || !(this.shouldLog('info'))) {
+  dir(...args) {
+    if (!args || !(this.shouldLog('info'))) {
       return;
     }
 
-    this.events.emit("log", "dir", obj);
-    this.logFunction(obj, null);
+    this.events.emit("log", "dir", args);
+    this.logFunction(args, null);
```

Comment:
I might be wrong here, but I'm not sure we want `...args` here. `dir()` should take an object/map.

Reply:
It's now correct to pass `args`.

See: https://github.com/embarklabs/embark/blob/fix/logger-dir/packages/core/logger/src/index.js#L26.

That was a refactor in my commit to PR #2184.